config-connector/solutions/projects/kpt/project-services/README.md

```diff
@@ -0,0 +1,42 @@
+Project Services
+==================================================
+
+# NAME
+
+  project-services
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a project.
+
+## Requirements
+  Your service account must have roles/serviceusage.serviceUsageAdmin or roles/owner
+
+
+## Usage
+  First, set the project-id:
+
+  ```
+  kpt cfg set . project-id VALUE 
+  ```
+
+  Before applying a service, set the service name and the project-id annotation. For example, to enable Compute Engine:
+
+  ```
+  kpt cfg set . service-name compute.googleapis.com
+  ```
+
+  Note: the package-default value will enable PubSub.
```

Comment:
And a similar link here would be great.

Reply:
Made it link to here: https://cloud.google.com/pubsub/docs